torchbenchmark/util/model.py

```diff
@@ -46,4 +47,52 @@ def _set_mode(self, train):
         (model, _) = self.get_module()
         model.train(train)
 
+    def check_results(self):
+        if not self.jit:
+            return
+
+        model_name = inspect.getfile(self.__class__)[-2]
+        model_blacklist = [
+            'demucs',
+            'yolov3',
+        ]
+
+        if model_name in model_blacklist:
+            warnings.warn(UserWarning(f"{model_name}.get_module() doesn't support `check_results` yet!"))
```

Comment:
is this just WIP (that you have both blacklist and try/except)? or you need both for some reason?

Reply:
@wconstab unfortunately, for some benchmarks results don't match, while the others are set up in a way I can get at their jitted module to be able to reset the optimization version and recompile it at noopt.
I added some comments. 
Eventually, I'm hoping to enable results checking for as many models as I can